pkg/controller/kfservice/kfservice_controller.go

```diff
@@ -132,30 +132,30 @@ func (r *ReconcileService) Reconcile(request reconcile.Request) (reconcile.Resul
 
 	serviceReconciler := ksvc.NewServiceReconciler(r.Client)
 	// Reconcile configurations
-	desiredDefault, desiredCanary := resources.CreateKnativeConfiguration(kfsvc)
```

Comment:
Can you leave all names in this file? Config is inferred in context.

Reply:
Done

Reply:
Is there a commit missing? I see see "desiredDefaultConfiguration"

Reply:
it's my mistake, I refactor the commit and push it by force